var/spack/repos/builtin/packages/powerapi/package.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack import *
+
+
+class Powerapi(AutotoolsPackage):
+    """This software is a reference implementation of the PowerAPI"""
+
+    homepage = "https://powerapi.sandia.gov/"
+    git      = "https://github.com/pwrapi/pwrapi-ref.git"
+
+    version('1.1.1', commit='93f66dfa29f014067823f2b790a1862e5841a11c')
+
+    variant('hwloc', default=False, description='Build hwloc support')
+    variant('debug', default=False, description='Enable debug support')
+    variant('mpi',   default=False, description='Enable MPI support')
+    variant('package', default=False, description='Enable package support')
+    variant('gnu-ld', default=False, description='Assume GNU compiled uses gnu-ld')
+
+    depends_on('autoconf')
+    depends_on('automake')
+    depends_on('libtool')
+    depends_on('m4')
+
+    depends_on('hwloc', when='+hwloc')
+    depends_on('mpi', when='+mpi')
+
+    def autoreconf(self, spec, prefix):
+        bash = which('bash')
+        bash('./autogen.sh')
+
+    def configure_args(self):
+        config_args = ['--prefix={0}'.format(self.prefix)]
```

Comment:
Adding `--prefix` is not needed ,please remove.

Reply:
If you rename `config_args` to `args`, maybe a bunch of stuff below can fit on one line.  (Also consider using `%` instead of `format()` for the same reason).

Reply:
Good call, I have made the requested changes. Thanks!